resources/assets/js/cachet.js

```diff
@@ -16,6 +16,9 @@ $(function () {
         beforeSend: function (xhr) {
             xhr.setRequestHeader('Accept', 'application/json');
             // xhr.setRequestHeader('Content-Type', 'application/json; charset=utf-8');
+          if (typeof window.apiKey !== 'undefined') {
```

Comment:
I really dislike this workaround. The problem originates from an API call that is fired whenever you open the dashboard.
Because the API is also "firewalled" you need the proper header in your request. Any tips on what to do with this?